packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -44,6 +48,11 @@ module.exports = function compileHtmlFiles( buildDir, manualTestScriptsPatterns
 	const staticFiles = _.flatten( sourceDirs.map( sourceDir => {
 		return globSync( path.join( sourceDir, '**', '*.!(js|html|md)' ) );
 	} ) ).filter( file => !file.match( /\.(js|html|md)$/ ) );
+	const languagesToLoad = [];
+
+	if ( options.additionalLanguages ) {
+		languagesToLoad.push( options.language, ...options.additionalLanguages );
```

Comment:
Does it work? There're two options:

1. Additional languages are defined, then:
   - The main language is added to the bundle
   - Other languages are added to separate files.
2. Additional languages aren't defined, then:
   - The main language is built-in to the bundle. No other files are generated.

I mean, these two lines aren't necessary IMO. I'd just set `const languagesToLoad = options.additionalLanguages || [];`

Reply:
There's also the `additionalLanguages: all` thing, that the `CKEditroWebpackPlugin` accepts, but IDK if we want to support it as the languages are determined during the compilation time based on the existing PO files.

Reply:
It is necessary. If both `language` and `additionalLanguages` are specified, you must load the former and the later. Otherwise

```
yarn manual --files=core,utils --language=en --additionalLanguages=ar,pl
```

<img width="860" alt="image" src="https://user-images.githubusercontent.com/1099479/60816742-a2361680-a19a-11e9-8c90-0ecb3063bec5.png">

because `translations/en.js` is missing

<img width="912" alt="image" src="https://user-images.githubusercontent.com/1099479/60816805-c5f95c80-a19a-11e9-8959-5a9ae4ec8644.png">

Reply:
Not sure why it is so. Can you investigate that?

Reply:
With the following config:

```js
new CKEditorWebpackPlugin( {
	// See https://ckeditor.com/docs/ckeditor5/latest/features/ui-language.html
	language: options.language,
	additionalLanguages: options.additionalLanguages,
	verbose: true
} )
```

I could remind myself the problem 😄

```
Warning: Because of the many found bundles, none of the bundles will contain the main language.
You should add it directly to the application from the 'translations/en.js'.
```

So... Actually, everything should be fine as it is now...